### DISCUSSION TITLE
Truenas Scale App - cannot mount volume correctly

### DISCUSSION THREAD

**Discussion by PrivateerGhosty** [photoprism/photoprism#2719] [Q&A]
Hello, all.

I'm trying to set up photoprism in TrueNas Core TrueNAS-SCALE-22.02.0.1 - official App, but I am not sure how to setup the volumes. Currently I have set up the `/mnt/something` folder into `/photoprism/originals`.

When I enter the shell in the pod, I see the pictures only if I mount them in `/originals` but not in `/photoprism/originals`.  I entered shell, with photos mounted in `/storage` and tried to mount `/storage` to `/photoprism/originals` and it said
`mount: /photoprism/originals: cannot mount /storage read-only`.
So, obviously when I mount the volume from TrueNas Core into photoprism it is read only, or probably `photoprism` folder in the pod is read only. Any idea hot to change that?

What am I doing wrong?

Thanks in advance

PS: I cannot use user 0, root. Invalid value. Must be greater than or equal to 1. The current user I use is 1000 and is with the correct right over this folder. Tested with other apps.

**Comment by graciousgrey** [comment]
Hi

Please note the TrueNAS app is no maintained by us.  If no TrueNAs user can help out here, you might want to contact the TrueNAS support :)

Please note the storage folder must be writable and not be located within originals. You can read more on volumes here https://docs.photoprism.app/getting-started/docker-compose/#volumes
